requirements/requirements-dev.txt

```diff
@@ -163,7 +163,7 @@ prometheus-client==0.20.0
     # via -r requirements.txt
 protobuf==3.20.3
     # via -r requirements.txt
-psutil==5.9.8
```

Comment:
Relevant change for Karapace is `psutil.connections()` rename to `psutil.net_connections()`. The new name is already used and is used only in tests. Major version bump is safe.